### DISCUSSION TITLE
Docker版无法生成频道！Help！！！

### DISCUSSION THREAD

**Discussion by tianniao886** [Guovin/iptv-api#713] [Q&A] [answered]
新拉取的guovern/iptv-api:latest。运行后无法生成频道，是哪里设置错了吗？？？
![1](https://github.com/user-attachments/assets/cc0c29be-4d84-48dc-a0ba-3d02150aae4a)
![2](https://github.com/user-attachments/assets/50252e11-119b-437f-ae11-a0f938195837)
![3](https://github.com/user-attachments/assets/69468e96-9fd5-435c-bfde-0f1b85ddfc88)

**Comment by Guovin** [comment]
确定是今天拉取的镜像吗？看日志似乎是旧版本。

**Comment by tianniao886** [comment]
新拉取的镜像。已经发现的问题。config中open_filter_speed = False 或 min_speed = 0.01可以生成ipv6频道。min_speed = 0.1会有几个频道或者没有。min_speed = 0.2时不会有任何IPV6频道生成。ipv4频道却不受此影响。何原因？截图生成的是ipv6频道。ipv4设置了关闭。
![4](https://github.com/user-attachments/assets/4c84834b-6463-4f70-b040-909225f4a41d)
![5](https://github.com/user-attachments/assets/1ada9fca-531f-4fc9-bc1b-bf095abf3dc4)

**Comment by Guovin** [comment]
你用的是旧镜像，新镜像已经修复该问题

**Comment by tianniao886** [comment]
这就是新拉取的镜像啊...还有其他版本吗？
![7](https://github.com/user-attachments/assets/d0dcea5d-37af-4978-9d1e-d42ea033ef3b)

**Comment by Guovin** [comment]
上面的日志是这个版本的吗？重新运行，还是有问题把日志发一下

**Comment by tianniao886** [comment]
日志就是这个版本的啊...min_speed = 0.01，可以生成ipv6频道。参数改0.02都不行...你看下频道排序，第一行355ms，第二行76ms，这个接口就是fmml_ipv6.txt源的。
![9](https://github.com/user-attachments/assets/e047ad50-fdc6-4c51-9701-59b06872e55c)

**Comment by tianniao886** [comment]
min_speed = 0.01 早晨运行时一个ipv6频道也没有生成
![10](https://github.com/user-attachments/assets/c4c2168a-379a-4f9b-8e8a-e2718c4f7823)

**Comment by tianniao886** [comment]
open_filter_speed = False 只能关闭了测速，手动运行生成ipv6频道.sort_timeout = 1~10.均对筛选ipv6频道不起作用。前一版是测试接口响应时间，还能筛选ipv6频道。
![11](https://github.com/user-attachments/assets/0fd7af72-4651-49bf-b008-1a97ab833a06)

**Comment by Guovin** [comment]
收到，我会尝试去复现。提醒一下，问题描述请总结好一次性发送，不要短时间分多次发送，频繁的邮件提醒会对关注者造成打扰，谢谢。

**Comment by Guovin** [comment]
该问题已经修复，重新拉取最新镜像。

**Comment by tianniao886** [comment]
已拉取Docker新版，已发现的问题：1）白名单中删除的频道，再次运行后生成频道依旧存在，不会被替换。2）sort_timeout = 1或10对生成的频道质量基本没啥影响。是不是可以设置个接口响应时间与接口速度权重比参数更好获取高质量频道？3）有时候运行中会卡住，只能手动重启。4）有的m3u源不能读取？如： 
 https://cdn.jsdelivr.net/gh/suxuang/myIPTV@main/ipv6.m3u
![1112](https://github.com/user-attachments/assets/91adbf6b-4994-41a8-b688-740b0d48d9f9)

**Comment by Guovin** [comment, marked as answer]
1.该问题已修复，等镜像更新
2.sort_timeout会影响接口测速延迟，对频道质量有影响。旧版有分辨率和延迟权重设置，但最近的版本已经移除，这是因为决定频道质量的最大因素是速率，所以权重值设置的影响微乎其微
3.该问题已修复，等镜像更新
4.并不是不能读取，而是使用了CDN代理，国内访问有可能会失败
